contracts/arbos/builtin/ArbSys.sol

```diff
@@ -1,31 +1,59 @@
 pragma solidity >=0.4.21 <0.7.0;
 
+/**
+* @title Precompiled contract that exists in every Arbitrum chain at address(100), 0x0000000000000000000000000000000000000064. Exposes a variety of system-level functionality.
+ */
 interface ArbSys {
-    // Get ArbOS version number
+    /**
+    * @notice  Get ArbOS version number
+    * @return version number int
```

Comment:
Maybe should say that this is the internal version number identifying an ArbOS build.